autovmaf_generate_excel.py

```diff
@@ -42,19 +47,34 @@
 
 for jobname in jobnames:
     RESULT_FILE = f"./results/{jobname}.json"
-    if file_exists(RESULT_FILE):
+    if not file_exists(RESULT_FILE):
+        logger.warning(f'⚠️  Skipping "{RESULT_FILE}": Not found.')
+        continue
```

Comment:
Previously the whole job would fail here, but I think it's better to try to continue with the other job names.